egs/gigaspeech/ASR/prepare.sh

```
@@ -138,7 +145,7 @@ if [ $stage -le 0 ] && [ $stop_stage -ge 0 ]; then
   #
   #   ln -svf /path/to/musan $dl_dir/
   #
-  if [ ! -d $dl_dir/musan ]; then
+  if [ $use_musan == true ] && [ ! -d $dl_dir/musan ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)